src/usernotifications.cs

```diff
@@ -394,6 +451,23 @@ interface UNNotificationContent : NSCopying, NSMutableCopying, NSSecureCoding {
 		[Mac (10,15)]
 		[NullAllowed, Export ("targetContentIdentifier")]
 		string TargetContentIdentifier { get; [NotImplemented] set; }
+
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Export ("contentByUpdatingWithProvider:error:")]
+		[return: NullAllowed]
+		UNNotificationContent ContentByUpdatingWithProvider (IUNNotificationContentProviding provider, [NullAllowed] out NSError outError);
```

Comment:
Methods should be named with a verb, so something like this:

```suggestion
		UNNotificationContent Update (IUNNotificationContentProviding fromProvider, [NullAllowed] out NSError outError);
```

This is very different, but it's how it's named in Swift (except that it's called `updating` in Swift, which isn't the verb tense we use)